README.md

```diff
@@ -146,3 +146,18 @@ import { getAuthenticatorFromEnvironment } from 'ibm-cloud-sdk-core';
 // MY_SERVICE_PASSWORD=<password>
 const cp4dAuthenticator = getAuthenticatorFromEnvironment('my-service');
 ```
+
+## Logging
+
+Logging is supported through the use of [winston](https://github.com/winstonjs/winston). Logging is disabled by default but can be enabled by setting the environment variable `ENABLE_LOGGER` to `true`. The optional flags below are also enabled through environment variables.
+
+
+When disabled, the logger has the following configuration option:
+- `LOG_TO_CONSOLE` - set to `true` if you wish to log messages to the console, but not to a log file. This is disabled by default.
+
+When enabled, the logger has a set of configuration options:
+
+- `LOG_LEVEL` - severity level of logging as outlined in the [winston docs](https://github.com/winstonjs/winston#logging). This is set to `info` by default.
+- `LOG_FILE_NAME ` - path of where to write the logs to. By default this is set to the current working directory `./logs/node-sdk-core.log`
+- `ENABLE_ROTATING_FILE` - set to `true` if you wish to [enable rotating file support](https://github.com/winstonjs/winston-daily-rotate-file). If enabled, this will write to the file specified by `LOG_FILE_NAME`. This is disabled by default.
+- `LOG_TO_CONSOLE` - set to `true` if you wish to also log messages to the console. Enabling this while the logger is enabled will log both to a file and the console. This is disabled by default.
```

Comment:
Reading through this again, there may be a simpler way to configure this. Can we do something like:
- Logger is off by default
- if `LOG_TO_CONSOLE` is defined, add a console transport
- if `LOG_FILE_NAME` is defined, add a file transport
- Rotating daily file and log level work the same way

That way we don't have to rely on the `ENABLE_LOGGER` flag as an extra variable the user needs to set. What do you think?

Reply:
I've gone ahead and updated the readme to follow something similar to what you suggested.